lib/ansible/modules/cloud/amazon/ec2_group.py

```diff
@@ -133,6 +133,8 @@
         group_desc: other example EC2 group
 '''
 
+import ipaddress
```

Comment:
I believe this module is only available in 2.7 and 3.3+, for modules using boto we support back to 2.6. How is boto doing this validation on Python 2.6? Can you copy them?